conjur/argument_parser/_login_parser.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, title_formatter
+
+
+class LoginParser:
+    def add_login_parser(self):
+        """
+        Method adds login parser functionality to parser
+        """
+        login_subparser = self._init_login_parser()
+        LoginParser._add_login_options(login_subparser)
```

Comment:
could this be `self._add_login_options(login_subparser)`?

Reply:
Changed